localization/italian/event_localization/immortality_set_age/major_decisions_middle_east_l_italian.yml

```diff
@@ -0,0 +1 @@
+﻿
```

Comment:
This one looks unfinished. Were you planning to do Italian in a separate PR? If so, we should remove it from this one.

I'm happy to have both in the same PR, but in that case the Italian localization must also be complete.

Reply:
ok i will complete the italian localization

Reply:
I just finished the italian localization, take a look. : )